fsspec/spec.py

```diff
@@ -228,6 +228,10 @@ def current(cls):
             return cls._cache[cls._latest]
         return cls()
 
+    @property
+    def transaction_type(self):
+        return Transaction
```

Comment:
How about a class attribute? Then it could be overridden either by any subclass having a different attribute or by setting the attribute on any instance.

Reply:
Changed, see [0271589](https://github.com/fsspec/filesystem_spec/pull/1424/commits/0271589c1f767098155e74f7a4fd5bdd1d1d90e4).